standalone/install.ts

```diff
@@ -0,0 +1,112 @@
+"use strict";
```

Comment:
Any reason we have the `"use strict";` here but not in other files? We should probably also consider the use of the  strict flag on the compiler also.

Reply:
ah, it was originally a javascript file.  I need to remove that.  typescript doesn't need strict :P